.env.example

```diff
@@ -22,7 +22,6 @@ MICROSOFT_CLIENT_SECRET=👻
 
 # AWS credentials for uploading user files from local and pull request environments to a staging S3 bucket
 AWS_S3_REGION=eu-west-2
-AWS_S3_ACL=public-read
```

Comment:
This env var is actually static and never change, I've simply removed it and hardcoded it here https://github.com/theopensystemslab/planx-new/pull/3989/files#diff-673b129524b241afeec398bcfbbc214d9df07c53667a53c3291890f6bca12acaR79